client/Contexts/ConfigFeatureFlag/Context.tsx

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright Strimzi authors.
+ * License: Apache License 2.0 (see the file LICENSE or http://apache.org/licenses/LICENSE-2.0.html).
+ */
+import React, { createContext, FunctionComponent, useState } from 'react';
+
+// Temporary config/feature flag definition - to be filled out in later PR with server calls. Add values here and in `strimzi-ui/config` directory
+const config = { example: { value: true } };
+const featureFlags = { capability: { enabled: false } };
+
+// Loading/Error/Trigger Refetch etc
+const contextStates = {
+  loading: false,
+  error: false,
+  isComplete: true,
+  triggerRefetch: () => true,
+};
+
+const ConfigFeatureFlag = createContext({
+  config: {},
+  featureFlags: {},
+  ...contextStates,
+});
+const ConfigFeatureFlagConsumer = ConfigFeatureFlag.Consumer;
+const ConfigFeatureFlagProvider: FunctionComponent = ({
+  children,
+  ...others
+}) => {
+  // get the retrieved config
+  const [configInState] = useState({ config, featureFlags });
+
+  return (
+    <ConfigFeatureFlag.Provider
+      {...others}
+      value={{ ...configInState, ...contextStates }}
+    >
+      {children}
+    </ConfigFeatureFlag.Provider>
+  );
+};
+
+export { ConfigFeatureFlagProvider, ConfigFeatureFlagConsumer };
```

Comment:
I'm not 100% this is always the case but the React hooks docs seems to suggest that the full context needs to be used for the useContext hook https://reactjs.org/docs/hooks-reference.html#usecontext

Reply:
> The React hooks docs says that the full context needs to be used for the useContext hook so can that be exported too. https://reactjs.org/docs/hooks-reference.html#usecontext

Indeed - In the next PR, I will have a `useConfigFeatureFlag` hook (which wraps `useContext`) so the context itself is private, but still accessible

Reply:
Ok perfect! As long as it's accessible through a hook